.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -476,20 +473,16 @@ dynamodb_GetItem:
       versions:
         - sdk_version: 3
           github: javascriptv3/example_code/dynamodb
-          sdkguide: sdk-for-javascript/v3/developer-guide/dynamodb-example-dynamodb-utilities.html#dynamodb-example-document-client-get
           excerpts:
             - description: Create the client.
               snippet_tags:
-                - dynamodb.JavaScript.tables.createclientv3
-            - description: Get an item from a table.
-              snippet_tags:
-                - dynamodb.JavaScript.item.getItemV3
+                - dynamodb.JavaScript.scenario.basics.createclientv3
             - description: Create the &DDB; document client.
               snippet_tags:
-                - dynamodb.JavaScript.tables.createdocclientv3
-            - description: Get an item from a table using the &DDB; document client.
+                - dynamodb.JavaScript.scenario.basics.createdocclientv3
+            - description: Put an item in a table.

```

Comment:
Put -> Get
(this is the GetItem excerpt)

Reply:
done